pom.xml

```diff
@@ -1620,6 +1620,104 @@
                 </plugins>
             </build>
         </profile>
+		<profile>
+			<id>java8</id>
+			<activation>
+				<jdk>1.8</jdk>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-enforcer-plugin</artifactId>
+						<version>3.0.0-M3</version>
```

Comment:
There shouldn't be a version here (and anywhere below), as this should be coming from the `<pluginManagement/>` section.